### ISSUE TITLE
Document matched uid does not exist in personnel table

### ISSUE THREAD

**Issue by huy-d-nguyen** [ppact/extraction#15]
Hi @tarakc02, we have checked the `documents` table in WRGL hub and it looks like some officers in the `documents` table do not exist in `personel` table. For example, the document with the `docid` **3c82a768** has the `matched_uid` **deda497a28be3d134292921ac546b3c4** but we cannot find that one in the `personnel` table. Can you help us check it again?

**Comment by huy-d-nguyen** [comment]
cc @rajivsinclair

**Comment by tarakc02** [comment]
Hi @huy-d-nguyen, I will take a look at it this afternoon and update this issue as soon as I know what's going on.

**Comment by tarakc02** [comment]
Hi everyone: that uid was in the personnel table when I last updated the documents table on Dec. 21 ([this is the snapshot I used](https://hub.wrgl.co/@ipno/r/personnel/commits/5898031bda1eb4a6e71d973963961f89)). I will run an update of the extraction/matching code this weekend, so only uid's that are in the latest snapshot are included. I had previously thought I would only need to run updates when I processed additional documents or made other changes, I did not realize that created uid's could change or disappear. I'll update my workflows accordingly.
